README.md

```diff
@@ -61,46 +61,32 @@ The Sympa server parameters can be customized in the _environments/local/group_v
 
 # Group vars for sympa servers
 
-# FQDN for mail addresses
-sympa_mail_hostname: lists.example.com
-
-# FQDN of web interface
-sympa_web_hostname: "{{ sympa_mail_hostname }}"
-
-# Web server admin email
-server_admin_email: "listmaster@{{ sympa_mail_hostname }}"
-
-# Root directory to install Sympa software
-sympa_root_directory: /home/sympa
-
-# Email addresses of listmasters (comma separated)
-sympa_listmaster_email: guevara@chemail.com
-
-# Sympa database type (either "mysql" or "Pg")
-sympa_db_type: "mysql"
-
-# Root passwd for the Sympa database
-sympa_db_root_password: "{{ lookup('file',inventory_dir+'/password/sympa_db_root_password') }}"
-
-# App passwd for the Sympa database
-sympa_db_app_password: "{{ lookup('file',inventory_dir+'/password/sympa_db_app_password') }}"
-
-# SSL cert for Apache
-sympa_ssl_certificate: "{{ lookup('file',inventory_dir+'/ssl_cert/webserver.crt') }}"
-
-# SSL private key for Apache
-sympa_ssl_key: "{{ lookup('file',inventory_dir+'/ssl_cert/webserver.key') }}"
+sympa:
+  db:
+    type: mysql
+    app_user: sympa
+    app_password: "{{ lookup('file',inventory_dir+'/private/password/sympa_db_app_password') }}"
```

Comment:
Could we not use a variable here and do the look up as a default for the variable

Reply:
Sorry, I do not understand what you want to do.
Because db.app_password is a variable.
Do you want to write something like that? 

`  db:
    type: mysql
    app_user: sympa
    app_password: sympa_app_password_content`

and somewhere else:

`sympa_app_password_content:  "{{ lookup('file',inventory_dir+'/private/password/sympa_db_app_password') }}"`

Is it your idea?

Reply:
yeah perfect it allows to supply a password without putting it in a file

Reply:
As for the webserver issue, I'd like to introduce it in another PR.? Because, in order to make it correctly, We should also modify the script generating the passwords, encrypted by the vault.
I can create a distinct issue if you like.